src/components/DataBar/DataBar.stories.tsx

```diff
@@ -130,5 +130,8 @@ export const InteractiveExample: StoryObj<Args> = {
     snapshot: {
       skip: true,
     },
+    axe: {
+      disabledRules: ['label'],
+    },
```

Comment:
Can you explain this change?

Reply:
The axe error associates with the textinput fields used to identify how much to change the data bar by (lines 103 and 115 in this file). I will change so that the associated text is a label and link them rather than disabling the rule. Thanks for pointing this out!